client/src/document/ingredients/browser-compatibility-table/atoms/_bc-history-link.scss

```diff
@@ -23,7 +23,10 @@
       content: "";
       display: block;
       height: 16px;
+      left: 50%;
       margin: 0 auto;
+      position: absolute;
```

Comment:
I am probably blowing up the scope of this issue, but looking at the BCD's CSS I started to wonder why we need this pseudo-element here (and a lot of its styles here).
I started playing with deleting it and just putting a centered background-image on the `.bc-history-link`. I then also had to `transform: rotate(180deg);` the element itself and flip the border position but it looked like basically the same thing with a whole lot less CSS.

I probably missed edge cases there and it is not quite what this issue is about, so do with that what you will.

Reply:
Hmmm. Yeah, I had not really interrogated it. Let me have a look at it tomorrow and perhaps we can indeed clean up some.

Reply:
Addressed this. I bet there is a lot of things like this we can do to improve the BCD table CSS. I have never really touched it in the transition phase to Yari. It is such a tricky component sometimes but, I would love to dig into it and clean up/improve for sure. Thanks for bringing this to my attention Gregor.